prerendering.bs

```diff
@@ -426,6 +431,20 @@ Patch the [=navigate=] algorithm to allow the [=prerendering browsing context/ac
     1. Return.
 </div>
 
+Patch the [=navigate=] algorithm to allow delaying or discarding of a cross-origin navigations in a [=prerendering browsing context=] as follows:
+
+<div algorithm="navigate delay cross-origin patch">
+  In [=navigate=], append the following steps after the cross-origin redirect check (currently step 13.1):
```

Comment:
There is no step 13.1 of navigate. Maybe you mean "process a navigate fetch"?

Reply:
This is still an issue.

Reply:
Oh I was sure I'd fixed that. Sorry, resubmitting.